src/Views/Pages/About/About.js

```diff
@@ -31,6 +41,53 @@ export default () => {
+            Officiis evertitur intellegam pri ne, mea eu convenire mnesarchum. An has tale fabulas scribentur. Abhorreant intellegebat te quo, ne pro inani repudiandae consectetuer, duo eu falli doming mnesarchum. Aperiam ocurreret usu no, denique accusata eloquentiam ei quo. Has augue quaerendum ad. Nam ut nemore animal commune, movet urbanitas id duo, sale discere tractatos te vix. Sit tempor dolorum an, sed aperiri ponderum eloquentiam at.
+            Ad oportere cotidieque eos, persius vocibus rationibus has no. Id erat ancillae consequat quo, ut vis case sadipscing. In mei ludus cotidieque vituperatoribus, pri te vero recteque pertinacia. Te percipitur assueverit vis, eu tation aperiam qui, iriure latine sed et. Tation iriure corpora ius ne, eum probo oratio eu. 
+            </p>
+        </div>
+        <div className={clsx(classes.header)}>
+            <h2> RPI </h2>
+        </div>
+        <div className={clsx(classes.content)}>
+            <p> 
+            Lorem ipsum dolor sit amet, est no ludus repudiandae. Augue vidisse in vix, consul intellegam no vis, pro ornatus partiendo vulputate an. Eleifend qualisque no pro. Ei soluta percipit usu. Cu vocent blandit mei.
+            In sea soleat doming, in soluta invidunt omittantur sea. Meis adhuc imperdiet his an. Id quis nibh saperet sed, te mea velit patrioque scriptorem. Qui laoreet reprimique eu, no duis virtute his. Ex graecis scriptorem omittantur est.
+            Cu nusquam lucilius sit, ut tale eirmod epicurei eam. Ne labore indoctum appellantur vix. Cu mel quis sensibus assueverit, putant euripidis ut pro. His blandit adolescens adversarium te, ne congue omittam mel. Vel ut unum aperiam explicari.
+            Officiis evertitur intellegam pri ne, mea eu convenire mnesarchum. An has tale fabulas scribentur. Abhorreant intellegebat te quo, ne pro inani repudiandae consectetuer, duo eu falli doming mnesarchum. Aperiam ocurreret usu no, denique accusata eloquentiam ei quo. Has augue quaerendum ad. Nam ut nemore animal commune, movet urbanitas id duo, sale discere tractatos te vix. Sit tempor dolorum an, sed aperiri ponderum eloquentiam at.
+            Ad oportere cotidieque eos, persius vocibus rationibus has no. Id erat ancillae consequat quo, ut vis case sadipscing. In mei ludus cotidieque vituperatoribus, pri te vero recteque pertinacia. Te percipitur assueverit vis, eu tation aperiam qui, iriure latine sed et. Tation iriure corpora ius ne, eum probo oratio eu. 
+            </p>
+        </div>
+        <div className={clsx(classes.header)}>
+            <h2> Contributors </h2>
+        </div>
+        <div className={clsx(classes.content)}>
+            <div>Person 1</div>
+            <div>Person 2</div>
+            <div>Person 3</div> 
```

Comment:
Notice how every section has the same structure? You could use myList.map(i => (<div.../div>)) to ensure that every one was generated using the same code.